src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.qna.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class Answers implements Iterable<Answer> {
+    private final List<Answer> answers;
+
+    public Answers() {
+        this(new ArrayList<>());
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public Answers delete(NsUser loginUser) {
+        List<Answer> deletedAnswers = new ArrayList<>();
+        for (Answer answer : answers) {
+            deletedAnswers.add(answer.delete(loginUser));
+        }
+
+        return new Answers(deletedAnswers);
+    }
+
+    public void add(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public DeleteHistories toDeleteHistories(LocalDateTime time) {
```

Comment:
delete()와 toDeleteHistories()를 합치는 것도 괜찮아보이네요 😄 
(수정을 위한 리뷰는 아니오니 참고만 해주세요!!)
```suggestion
    public DeleteHistories delete(NsUser user, LocalDateTime time) {
```

Reply:
네, 해당 부분은 인지하고 넘어가곘습니다! 감사합니다